jenkinsfiles/ios_jenkinsfile.groovy

```diff
@@ -84,4 +92,4 @@ node('ios') {
     stage('Archive') {
         archiveArtifacts "build/Debug-iphoneos/${OUTPUT_FILE_NAME}"
```

Comment:
no reason at all, I can add this param in the plugin.

AFAIK The difference between debug and release builds is that debug builds will show more low level compiler logs when building an app + the artifact is a bit bigger than a release one.